src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -93,6 +95,14 @@ const modalScreenListeners = {
     },
 };
 
+const defaultScreenOptions = {
+    cardStyle: {
+        overflow: 'visible',
+    },
+    headerShown: false,
+    animationTypeForReplace: 'pop',
+};
```

Comment:
Seems like `defaultScreenOptions` is [also declared in `PublicScreens.js` with the same values](https://github.com/Expensify/Expensify.cash/blob/36a336400efc060df2ebc21d74ca1ee19ee5afff/src/libs/Navigation/AppNavigator/PublicScreens.js#L10-L16). Maybe they should be moved into a separate config file so we don't have to remember to update things twice in the future when applying a change?

Reply:
Sure, I can do that.